Tests/TestingTests/SwiftPMTests.swift

```diff
@@ -83,6 +91,28 @@ struct SwiftPMTests {
     #expect(!testFilter(test3))
   }
 
+  @Test("--xunit-output argument")
+  func xunitOutput() throws {
+    // Test that a bad path produces an error.
+    #expect(throws: CError.self) {
+      _ = try configurationForSwiftPMEntryPoint(withArguments: ["PATH", "--xunit-output", "/nonexistent/path/we/cannot/write/to"])
+    }
+
+    // Test that a file is opened when requested. Testing of the actual output
```

Comment:
This feels like it's really two tests. Maybe create a nested suite type and decompose these into two `@Test` funcs within?

Reply:
Done.